applications/GeoMechanicsApplication/tests/test_dynamics_long.py

```diff
@@ -17,55 +20,55 @@ def test_constant_strip_load_2d(self):
         """
         Tests a constant strip load on a 10m X 10m  block. The block is loaded with a constant strip load of 1kN/m and
         a length of 1m.
-
+    
```

Comment:
There have been added several spurious sequences of spaces. I guess that was done accidentally. Can you please revert them?